lib/console/devices/device_resolver.ex

```diff
@@ -117,7 +118,10 @@ defmodule Console.Devices.DeviceResolver do
     devices = Device
       |> where([d], d.organization_id == ^current_organization.id)
       |> Repo.all()
-      |> Enum.map(fn d -> Map.drop(d, [:app_key]) end)
+      |> Enum.map(fn d ->
+        Map.drop(d, [:app_key])
```

Comment:
you have to chain 122 and 123 together, otherwise app key doesn't get dropped from d

Reply:
check last commit

Reply:
wait not yet